client/src/components/Reviews.jsx

```diff
@@ -26,7 +103,7 @@ class Reviews extends React.Component {
 
   like(event) {
     const reviewId = {
-      reviewId: event.target.parentNode.parentNode.getAttribute('review_id'),
+      reviewId: event.target.parentNode.parentNode.getAttribute('id'),
```

Comment:
This feels complex, though I don't doubt it works - can you manage this through state at all?

Reply:
Yeah that's a good idea. I could refactor to have a review subcomponent with an internal state for the `reviewId` & `likes_count`...